serverlessworkflow/modules/ROOT/pages/integrations/serverless-dashboard-with-runtime-data.adoc

```diff
@@ -1 +1,315 @@
+                    group:
+                      - columnGroup:
+                          source: labels
+                        groupFunctions:
+                          - source: labels
+                          - source: value
+      - columns:
+          - span: "6"
+            components:
+              - html: >-
+                  <p style="font-size: large">Completed </p>
+              - settings:
+                  type: BARCHART
+                  chart:
+                    resizable: 'true'
+                    width: '2000'
+                  dataSetLookup:
+                    uuid: metrics
+                    sort:
+                      - column: value
+                        sortOrder: DESCENDING
+                    filter:
+                      - column: metric
+                        function: EQUALS_TO
+                        args:
+                          - "kogito_process_instance_completed_total"
+                    group:
+                      - columnGroup:
+                          source: labels
+                        groupFunctions:
+                          - source: labels
+                          - source: value
+          - span: "6"
+            components:
+              - html: >-
+                  <p style="font-size: large">Total Duration </p>
+              - settings:
+                  type: BARCHART                  
+                  chart:
+                    resizable: 'true'
+                    width: '2000'
+                  lookup:
+                    uuid: metrics
+                    sort:
+                      - column: value
+                        sortOrder: DESCENDING
+                    filter:
+                      - column: metric
+                        function: EQUALS_TO
+                        args:
+                          - "kogito_process_instance_duration_seconds_sum"
+                    group:
+                      - columnGroup:
+                          source: labels
+                        groupFunctions:
+                          - source: labels
+                          - source: value
+----
+
+.Workflow Metrics Dashboard
+image::getting-started/integrations/dashbuilder-workflows-metrics-dashboard.png[]
+
+If you are using Prometheus we can use PROMQL with Prometheus REST API and transform the result to the JSON format supported by Dashbuilder to build dashboards. For further information about Dashbuilder and Prometheus check the article https://blog.kie.org/2022/07/creating-prometheus-dashboards-using-dashbuilder.html[Creating Prometheus Dashboards with Dashbuilder].
+
+== Using Data Index
+
+Data index is a service available during development mode and it contains all data about the workflow steps execution. By default data index is available at `http://localhost:8180` and it uses GraphQL, so Dashbuilder can connect to it using its REST API and transform the result to the supported format.
```

Comment:
```suggestion
Data index is a service available during development mode, containing all data about the execution of workflow steps. By default, the data index is available at http://localhost:8180, and it uses GraphQL, so Dashbuilder can connect to it using its REST API and transform the result to the supported format.
```

You can leave the link since the user might be reading this while working. It will make it easier for them to just click on it.